alf/algorithms/data_transformer.py

```diff
@@ -922,4 +1022,7 @@ def create_data_transformer(data_transformer_ctor, observation_spec):
     if len(data_transformer_ctor) == 1:
         return data_transformer_ctor[0](observation_spec)
 
+    if HindsightExperienceTransformer in data_transformer_ctor:
+        assert HindsightExperienceTransformer == data_transformer_ctor[0], \
+            "Hindsight relabeling should happen before all other transforms."
```

Comment:
This is no longer true after Break's recent PR on `SequentialDataTransformer`. Basically `UntransformedTimeStep` could appear before any other data transformer. I suggest either justifying why `HindsightExperienceTransformer` has to appear before `UntransformedTimeStep` or removing this assertion here.